.travis.yml

```diff
@@ -17,7 +17,10 @@ branches:
 env:
   # Keep this Bazel version in sync with the `versions.check` directive
   # near the top of our WORKSPACE file.
-  - BAZEL=0.16.1 TF=NIGHTLY
+  #
+  # Grab the BAZEL_SHA256SUM from the Bazel releases page; e.g.:
```

Comment:
I added this primarily to protect against the case where the GitHub
release differs from the Bazel mirror. This _shouldn’t_ happen, as far
as I know, but if it did then it could lead to really confusing
failures; for instance, a failed build could pass when re-run.

(In general, I think that _anything_ that downloads a known binary and
then executes it should be verified for integrity. There’s just no
reason not to. :-) )

> I'm not exactly sure what there's to gain maliciously from mucking
> with our CI build.

Right; there shouldn’t be anything. Travis doesn’t expose secret
environment variables to PRs from forks for this reason, but we don’t
have any such environment variables, anyway.

Reply:
The reason not to is the extra hassle factor and the fact that (IMO) it doesn't add anything to the security properties of the travis run which already is executing arbitrary code on purpose - but if you feel strongly about it sure.